tests/azure/availset_test.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 func genAvailSetName() string {
-	return fmt.Sprintf("klb-availset-tests-%d", rand.Intn(1000))
+	return fmt.Sprintf("klb-availset-tests-%d", rand.Intn(9999999))
```

Comment:
Yeah...very sad for this. Any tips on a cool random uuid ? I remember you gave me one once, but I lost it :-(